.github/workflows/main.yml

```
@@ -106,7 +106,7 @@ jobs:
           # MacOS runner has way too much preinstalled Python versions. We must
           # use the exact one discovered by CMake.
           PYTHON=`cat build/CMake_Python3_EXECUTABLE`
-          PYTHONPATH=$PYTHONPATH:$PREFIX/$($PYTHON -c "import distutils.sysconfig as sc; print(sc.get_python_lib(prefix='', plat_specific=True,standard_lib=False))")
+          PYTHONPATH=$PYTHONPATH:$PREFIX/$($PYTHON -c "import sysconfig as sc; print(sc.get_path('platlib', sc.get_preferred_scheme('user'), vars={'userbase': '.'}))")
```

Same comment as on the fontforgebuilds pr, I think this should be

```suggestion
          PYTHONPATH=$PYTHONPATH:$PREFIX/$($PYTHON -c "import sysconfig as sc; print(sc.get_path('platlib', sc.get_preferred_scheme('prefix'), vars={'platbase': '.'}))")
```

---

Please see https://github.com/jtanx/fontforgebuilds/pull/21#issuecomment-2131031460

> Unfortunately `'prefix'` key is inconsistent across platforms. On MacOS the preferred scheme for `prefix` is `osx_framework_library`, and its platlib entry looks like
> 
> ```
> 'platlib': '/usr/local/{platlibdir}/python{py_version_short}/site-packages'
> ```
> 
> This forces absolute path (which is discouraged by CMake) and the variable names are different for each platform. You can check this at https://github.com/iorsh/fontforge/actions/runs/9223726161/ - see the "Run pyhook smoke test" entry for Mac and Linux